### ISSUE TITLE
Getting RunTime error on install

### ISSUE THREAD

**Issue by thornomad** [SublimeLinter/SublimeLinter-flake8#22]
I'm trying to install via Package Control; I first install SublimeLinter 3.  Then installed SublimeLinter-flake8.  I get this error in the console:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 74, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 678, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "linter in /Users/thornomad/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter-flake8.sublime-package", line 17, in <module>
  File "/Users/thornomad/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 124, in __init__
    cls.reinitialize()
  File "/Users/thornomad/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/python_linter.py", line 98, in reinitialize
    cls.initialize()
  File "linter in /Users/thornomad/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter-flake8.sublime-package", line 74, in initialize
RuntimeError: super(): empty __class__ cell
```

This is my first time using SublimeLint and am not sure what to do ... it doesn't appear to be working.  Have tried this on two different machines.  After this error I don't see `SublimeLinter-flake8` in my **Library/Application Support/Sublime Text 3/Packages**.  And every file I check it says "no errors".

Thanks for your help.

**Comment by skj3gg** [comment]
Hey @thornomad,

Sorry you're having issues. First `SublimeLinter-flake8` should be found in **Library/Application Support/Sublime Text 3/Installed Packages**, so hopefully you find it in there... 

Can you give me the version of SublimeText you are using and the full console output on a fresh start of SublimeText.

**Comment by thornomad** [comment]
Thanks for getting back to me so quick!

To answer your questions:

* Yes, `SublimeLinter-flake8.sublime-package` is present in `~/Library/Application Support/Sublime Text\ 3/Installed Packages`

Second, the version reported is:

    startup, version: 3083 osx x64 channel: stable

You can see the whole console output here:  <http://pastebin.com/ZcrWdLBr> 

But I noticed two things myself:

    reloading plugin SublimeLinter-flake8.linter
    SublimeLinter: flake8 linter loaded
    [...]
    SublimeLinter: debug mode: off 
    SublimeLinter: WARNING: import of flake8.engine module in flake8 failed, linter will not work with python 3 code

And I also I noticed that sublime is searching through all my .tox directories ... not sure if that's cool or not.

Thanks!

**Comment by snario** [comment]
+1

**Comment by snario** [comment]
@thornomad any fix?

**Comment by thornomad** [comment]
So for me I believe the problem had to do with my python pointing to python3 ... At least I think that's what it was. Sorry. Should have replied when I fixed it.

**Comment by snario** [comment]
So 'which python' pointed to Python 3 or was there a setting within sublime?

**Comment by thornomad** [comment]
Which Python ... It was a global issue not a sublime based one. I had
gotten screwy by installing multiple versions of Python ...

**Comment by romanlv** [comment]
The same problem on Linux Mint 17.3 

```
reloading plugin SublimeLinter-flake8.linter
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 78, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text/sublime_plugin.py", line 696, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "linter in /home/roman/.config/sublime-text-3/Installed Packages/SublimeLinter-flake8.sublime-package", line 17, in <module>
  File "/home/roman/.config/sublime-text-3/Packages/SublimeLinter/lint/linter.py", line 123, in __init__
    cls.reinitialize()
  File "/home/roman/.config/sublime-text-3/Packages/SublimeLinter/lint/python_linter.py", line 97, in reinitialize
    cls.initialize()
  File "linter in /home/roman/.config/sublime-text-3/Installed Packages/SublimeLinter-flake8.sublime-package", line 76, in initialize
RuntimeError: super(): empty __class__ cell
```
`which python` references python 2.7, although python3 is installed as well
Sublime Text build `3103`

Installing individual `sublimelinter-pep8` and `sublimelinter-pyflake` works fine

**Comment by groteworld** [comment]
looks like OP was fixed and pathing issues should be fixed now in v3.0.0
